core/templates/services/math-interactions.service.ts

```diff
@@ -81,11 +82,23 @@ export class MathInteractionsService {
         'Your answer has two symbols next to each other: "' + symbol1 +
         '" and "' + symbol2 + '".');
     }
+    if (
```

Comment:
Can you post a screenshot of the new error message showing?

Reply:
I've now added it to the PR description.